scripts/jenkins/jobs-ibs/cloud-mkcloud7.yaml

```diff
@@ -45,7 +45,15 @@
     arch: s390x
     label: openstack-mkcloud-SLE12-{arch}
     jobs:
+        - 'cloud-mkcloud{version}-job-backup-restore-{arch}'
+        - 'cloud-mkcloud{version}-job-btrfs-{arch}'
         - 'cloud-mkcloud{version}-job-crowbar_register-{arch}'
+        - 'cloud-mkcloud{version}-job-dvr-{arch}'
+        - 'cloud-mkcloud{version}-job-ha-{arch}'
```

Comment:
no, but none of those jobs work anyway since we don't have the necessary mkcloud fixes..